src/lessons/02_introduction_to_frontend/introduction-to-frontend.slides.html

```diff
@@ -0,0 +1,688 @@
+
+            <ul class="tags-with-code">
+              <li><code>&lt;div&gt;</code> defines a division or a section in an HTML document.</li>
+              <li><code>&lt;img&gt;</code> used to embed an image in an HTML page. <small style="vertical-align: middle;">(remember: there's not content within img)</small></li>
+              <li><code>&lt;a&gt;</code> defines a hyperlink, which is used to link from one page to another. (`a` as in anchor)</li>
+              <li class="fragment small-please" style="margin-top: 50px; color: gray;"><code>&lt;!-- SOME COMMENT --&gt;</code> specifies a comment in a page (only visible for developers).</li>
+            </ul>
+
+            <div class="fragment">
+              <p>👇🏼 The ones you know already</p>
+
+              <ul class="tags-with-code">
+                <li><code>&lt;p&gt;</code> represents a paragraph.</li>
+                <li><code>&lt;h1&gt;, &lt;h2&gt; ... &lt;h6&gt;</code> used to define HTML headings, from most to less important.</li>
+              </ul>
+            </div>
+          </section>
+
+          <!--SLIDE 8.4-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please"><b>Remind</b>: Some of those tags contain important attributes!</p>
+          </section>
+
+          <!--SLIDE 8.5-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please"><b>Remind</b>: Some of those tags contain important attributes!</p>
+
+            <p>"OK, but what are attributes of a tag?" 🤔💭</p>
+            <img data-src="images/curious.gif"/>
+          </section>
+
+          <!--SLIDE 8.5-->
+          <section data-auto-animate data-timing="10">
+            <h3>More structural tags, for you</h3>
+            <p class="small-please" style="font-weight: bold;">👉 Attributes are special words used inside the opening tag to control the element's behaviour</p>
+
+            <ul class="tags-with-code">
+              <li><code>&lt;img&gt;</code> contains, for instance, the <code>src=""</code> attribute. <small style="vertical-align: middle;">(and many others)</small></li>
+              <li>whereas <code>&lt;a&gt;</code> contains <code>href=""</code> and <code>target=""</code>.</li>
+            </ul>
+
+            <br/><br/>
+
+            <ul class="fragment tags-with-code tags-with-code--small">
+              <li><code>src="YOUR_IMAGE.jpg"</code> will make the <b>img</b> tag displays the image you have selected.</li>
+              <li>whereas <code>href="http://somewhere.com"</code> will make the anchor clickable to the page you want the user to go.</li>
+              <li>and <code>target="_blank|_self"</code> will make the anchor open the href outside or inside the page user is currently viewing (`_self` is default).</li>
+            </ul>
```

Comment:
I feel like this needs a code example as well. Just a quick one, maybe one with `img src` and one with `a href` so that the students can see the entire syntax

Reply:
I was willing to have a demo at this point, wdyt?

But yeah, I can also add that I believe its great!

Reply:
I'm mainly thinking of students who look at the slides after the lesson again. Simple examples are enough. More complex stuff is for the demo

Reply:
Makes total sense. Thanks for the suggestion.